extensions/eclipse-che-theia-about/src/browser/about-che-theia-dialog.ts

```diff
@@ -10,46 +10,80 @@
 
 import { AboutDialog, AboutDialogProps, ABOUT_EXTENSIONS_CLASS, ABOUT_CONTENT_CLASS } from '@theia/core/lib/browser/about-dialog';
 import { injectable, inject, postConstruct } from 'inversify';
-import { ThemeService, Theme } from '@theia/core/lib/browser/theming';
-const logoDark = require('../../src/browser/style/che-logo-dark.svg');
-const logoLight = require('../../src/browser/style/che-logo-light.svg');
+import { CheProductService, ProductInfo } from '@eclipse-che/theia-plugin-ext/lib/common/che-protocol';
+import '../../src/browser/style/che-theia-about.css';
 const jsonDetails = require('../../conf/about-details.json');
 
 @injectable()
 export class AboutCheTheiaDialog extends AboutDialog {
 
-    @inject(ThemeService)
-    protected readonly themeService: ThemeService;
+    @inject(CheProductService)
+    private productService: CheProductService;
 
     constructor(
         @inject(AboutDialogProps) protected readonly props: AboutDialogProps
     ) {
-        super(props);
+        // use empty header by default
+        super({
+            title: ''
+        });
     }
 
-    @postConstruct()
-    protected async init(): Promise<void> {
-        const messageNode = document.createElement('div');
-        messageNode.classList.add(ABOUT_CONTENT_CLASS);
-        const imgObject = document.createElement('img');
+    /**
+     * Returns suitable URI as a string for the logo image
+     */
+    protected getLogo(logo: string): string {
+        if (logo.startsWith('http://') || logo.startsWith('https://')) {
+            // HTTP resource
+            return logo;
+        }
+
+        if (logo.startsWith('file://')) {
+            // file resource
+            logo = logo.substring(7);
+        }
 
-        if (this.isDark(this.themeService.getCurrentTheme())) {
-            imgObject.src = logoDark;
+        if (logo.startsWith('/')) {
+            return `/webview${logo}`;
         } else {
-            imgObject.src = logoLight;
+            return `/webview/${logo}`;
         }
+    }
 
-        // listen on events when the theme is changing to update the logo
-        this.themeService.onThemeChange(e => {
-            if (this.isDark(e.newTheme)) {
-                imgObject.src = logoDark;
-            } else {
-                imgObject.src = logoLight;
-            }
+    /**
+     * Returns product header element
+     */
+    protected async getProductHeader(product: ProductInfo): Promise<HTMLElement> {
+        const header = document.createElement('div');
+        header.setAttribute('class', 'che-theia-about-product-header');
 
-        });
```

Comment:
is it still changing from dark to light without refreshing the page if you do this ?

Reply:
One image is used for both themes. It looks well.

Reply:
hmm before we had two logos. One for dark theme, one for light theme

I think it's because here you're displaying the text 'Eclipse Che' from a string and not from the logo like https://github.com/eclipse/che/blob/master/dashboard/src/assets/branding/che-logo.svg which includes both logo and text.

Reply:
Yes, previous logo image had also the product name.
Now the product name displayed as a text using theme colors.